runtime/stdlib/crypto.go

```diff
@@ -43,8 +43,7 @@ var CryptoChecker = func() *sema.Checker {
 		location,
 		nil,
 		false,
-		sema.WithPredeclaredValues(BuiltinFunctions.ToSemaValueDeclarations()),
-		sema.WithPredeclaredTypes(BuiltinTypes.ToTypeDeclarations()),
+		// TODO: base values and types
```

Comment:
How come the type checking for Crypto contract works without the built-in types? (e.g: `PublicKey`, etc.)?

Reply:
Turns out the Crypto contract does not actually use any of the stdlib values. It does indeed use the `PublicKey` type – but not the value (constructor)! I'll remove this TODO.